apps/minaintyg/src/pages/Certificate/CertificatePage.tsx

```diff
@@ -25,6 +25,7 @@ export function CertificatePage() {
 
   return (
     <>
+      <ScrollRestoration />
```

Comment:
Not sure if this is the best way to solve it or if it should be moved, but it seems to work.

Reply:
If it works it seems like a good fix! Did you check so you don't get any errors in the console?